src/getConfig.js

```diff
@@ -72,7 +72,7 @@ function unknownValidationReporter(config, example, option, options) {
   will fix it and remove this message.`
 
     const comment = options.comment
-    const name = (options.title && options.title.warning) || 'WARNING'
+    const name = options.title && options.title.warning
```

Comment:
Hey @Andrew1024, could you please tell me the reason for this change?

Reply:
The options.titile and options.title.warning gets defaulted inside `jest` , so we don't need this redundant check.

It gets defaulted [here](https://github.com/facebook/jest/blob/5e903c1f31d8db8d914b7ae84830ad90c73061fc/packages/jest-validate/src/validate.js#L55)

Reply:
So could we just change it to `const name = options.title.warning`?

Reply:
Yeah, I didn't think about that